cmd/clusterctl/client/config/providers_client.go

```diff
@@ -49,6 +49,7 @@ const (
 	// ControlPlane providers
 	KubeadmControlPlaneProviderName = "kubeadm"
 	TalosControlPlaneProviderName   = "talos"
+	EKSControlPlaneProviderName     = "eks"
```

Comment:
```suggestion
	AWSEKSControlPlaneProviderName     = "aws-eks"
```

?

Reply:
@vincepri  - that is more explicit, i like it. Should we also rename the [EKS boostrap provider](https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/clusterctl/client/config/providers_client.go#L47) which is already merged? So that it becomes:

```go
AWSEKSBootstrapProviderName     = "aws-eks"
```

/cc @mytunguyen @rudoi

Reply:
I would say yes, for consistency.

Reply:
Sounds good to me, fine to change now given that the related commit https://github.com/kubernetes-sigs/cluster-api/commit/b068d9498b9ff060a686515758444c29f1722363 hasn't been released yet

Reply:
@vincepri  @fabriziopandini - Changing the existing EKS boostrap provider is causing issues with the apidiff test.

```
Incompatible changes:
  - EKSBootstrapProviderName: value changed from "eks" to "aws-eks" 
```

Reply:
I've kept the old provider name to satisfy the apidiff tests and added the new provider name. They both return the same metadata and the provider list only includes the new aws-eks bootstrap provider

Reply:
The test is optional, you should be good!